app/views/admins/orders/orders.csv.erb

```diff
@@ -0,0 +1,13 @@
+<%=raw CSV.generate_line(["Status", "Order ID", "Reward ID", "Reward title", "Reward description", "Order creation time", "Update time", "Purchase price", "Employee ID", "Employee email"]) -%>
+<%- @orders.each do |order| -%>
+  <%=raw CSV.generate_line([order.delivery_status, 
+                         order.id, 
```

Comment:
formatting issue all `order.` should be aligned

Reply:
corrected 🙂